OpenTrons/opentrons-api
# fix(protocol-designer): cleanup and add clear functionality to LiquidToolbox

# Overview

This PR updates style according to latest designs and implements missing functionality in LiquidToolbox. Here, I add the ability to clear a specific liquid from its loaded wells from its liquid card, and update some other minor style issues. I also add an InfoScreen if there are no loaded liquids on the selected labware and no wells are selected.

Lastly, I add prevention of highlighting labware column and row labels when clicking and dragging

Closes RQA-3341
Closes RQA-3342

## Test Plan and Hands on Testing

https://github.com/user-attachments/assets/aca0304c-5ac9-4ba7-856f-698847374fec

- on protocol starting deck tab, select a labware without liquid, and select 'add liquid'
- verify that no liquids infoscreen shows in toolbox
- select some wells and verify that the add liquid box appears
- add some liquid to the selected wells
- select 'delete' from the new liquid card and verify that liquids are correctly removed from its wells

#### Click and drag fix

before fix:

https://github.com/user-attachments/assets/235a94e2-559e-40d1-b8c8-62fa7bf6d445

after fix:

https://github.com/user-attachments/assets/1cc727cc-aa5c-471e-9fe3-f5e80b501cca

## Changelog

- refine `InfoScreen` component and add to `LiquidToolbox` when liquids are empty and no wells are selected
- use `Tag` component in dropdown
- add delete liquid functionality to liquid card

## Review requests

see test plan

## Risk assessmen

low
